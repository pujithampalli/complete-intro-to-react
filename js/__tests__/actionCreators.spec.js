import { setSearchTerm, addAPIData } from "../actionCreators";

test("setSearchTerm", () => {
  expect(setSearchTerm("black")).toMatchSnapshot();
});

test("addAPIData", () => {
  expect(
    addAPIData({
      tt2085059: {
        rating: "0.0",
        title: "Black Mirror",
        year: "2011â€“",
        description: "A television anthology series that shows the dark side of life and technology.",
        poster: "bm.jpg",
        imdbID: "tt2085059",
        trailer: "jDiYGjp5iFg"
      }
    })
  ).toMatchSnapshot();
});
