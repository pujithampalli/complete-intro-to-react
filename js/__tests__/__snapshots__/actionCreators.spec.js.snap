// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addAPIData 1`] = `
Object {
  "payload": Object {
    "tt2085059": Object {
      "description": "A television anthology series that shows the dark side of life and technology.",
      "imdbID": "tt2085059",
      "poster": "bm.jpg",
      "rating": "0.0",
      "title": "Black Mirror",
      "trailer": "jDiYGjp5iFg",
      "year": "2011â€“",
    },
  },
  "type": "ADD_API_DATA",
}
`;

exports[`setSearchTerm 1`] = `
Object {
  "payload": "black",
  "type": "SET_SERCH_TERM",
}
`;
